twilio_am_temporary.json

```diff
@@ -0,0 +1,116 @@
+{
+  "openapi": "3.0.1",
+  "security": [
+    {
+      "oAuth2ClientCredentials": []
+    }
+  ],
+  "info": {
+    "title": "Organization Public API",
+    "contact": {
+      "name": "Twilio Support",
+      "url": "https://support.twilio.com",
+      "email": "support@twilio.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/Organizations/{OrganizationSid}": {
+      "servers": [
+        {
+          "url": "https://preview-iam.twilio.com"
+        }
+      ],
+      "get": {
+        "summary": "List SCIM Users",
```

Comment:
List Organizations?

Reply:
Yes, We can modify these changes. Also please check what should be the error.
I have just made sure that 2 files get merged only in python language.